dask/tests/test_order.py

```diff
@@ -232,16 +232,22 @@ def test_order_doesnt_fail_on_mixed_type_keys(abcde):
 def test_gh_3055():
     da = pytest.importorskip('dask.array')
     A, B = 20, 99
-    x = da.random.normal(size=(A, B), chunks=(1, None))
+    orig = x = da.random.normal(size=(A, B), chunks=(1, None))
     for _ in range(2):
         y = (x[:, None, :] * x[:, :, None]).cumsum(axis=0)
         x = x.cumsum(axis=0)
     w = (y * x[:, None]).sum(axis=(1,2))
 
     dsk = dict(w.__dask_graph__())
     o = order(dsk)
+    # dask.visualize(dsk, color='order', node_attr={'penwidth': '6'},
+    #                filename='dask.pdf')
```

Comment:
Should this be dropped?

Reply:
I've added these lines the last few times I've played with order.  I'm inclined to leave them in for myself or others.  The image made by this test is particularly valuable to check correctness.

Reply:
Generally don't find commented code to be a good idea to keep around (especially in a nice library like Dask). It is seldom updated or used for that matter. Also it raises questions for later users as to its value (at which point it may end up being scrapped anyways). Not to mention it makes the code base a bit uglier. If the code is simple like this, it is easy for anyone to add should it be needed. Would recommend dropping it. However if you feel strongly about keeping it, would encourage including it as part of the test explicitly (uncommented) so it doesn't fall victim to the aforementioned issues. Given how easy this is to solve, would encourage picking one of these two options and addressing it before merging.

Reply:
As you like.  Removed